webapi/src/main/scala/org/knora/webapi/Main.scala

```diff
@@ -52,6 +52,9 @@ object Main extends App with LiveCore with KnoraService {
     // starts zipkin monitoring reporter
     if (arglist.contains("-j")) applicationStateActor ! SetZipkinReporterState(true)
```

Comment:
Instead of parsing command-line arguments yourself, you could use a library. I've had good results with scallop:

https://github.com/scallop/scallop

Reply:
I know, I looked at Scallop, but it looked complicated and I didn't want to spend to much time on it. Though if I add something else in the future, I will change to scallop.